src/tests/JIT/opt/Vectorization/StringEquals.cs

```diff
@@ -195,10 +195,22 @@ static void ValidateEquals(bool result, string left, string right)
     [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_157(string s) => ValidateEquals(s == "2ж1311a23b2212aЙ21Й11жb3233bb3a1", s, "2ж1311a23b2212aЙ21Й11жb3233bb3a1");
     [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_158(string s) => ValidateEquals(s == "01Й11113ь3Й32a3ьЙЙ3Й32b2ab221310", s, "01Й11113ь3Й32a3ьЙЙ3Й32b2ab221310");
     [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_159(string s) => ValidateEquals(s == "a120213b11211\0223223312ьь1Й3222Й", s, "a120213b11211\0223223312ьь1Й3222Й");
+    [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_160(string s) => ValidateEquals(s == "鉄", s, "鉄");
+    [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_161(string s) => ValidateEquals(s == "鉄鉄", s, "鉄鉄");
+    [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_162(string s) => ValidateEquals(s == "鉄鉄鉄", s, "鉄鉄鉄");
+    [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_163(string s) => ValidateEquals(s == "\uFFFF", s, "\uFFFF");
+    [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_164(string s) => ValidateEquals(s == "\uFFFF\uFFFF", s, "\uFFFF\uFFFF");
+    [MethodImpl(MethodImplOptions.NoInlining)] public static void Equals_165(string s) => ValidateEquals(s == "\uFFFF\uFFFF\uFFFF", s, "\uFFFF\uFFFF\uFFFF");
 
     public static readonly string[] s_TestData =
     {
         null,
+        "鉄",
```

Comment:
Coding guidelines I think say to always use ASCII, ie use \uNNNN to avoid accidental garbling later. I don't know whether we are actually consistent. But in this case it would perhaps be clearer.

Reply:
There are some existing non-ASCII chars in this file:
https://gist.github.com/am11/dddd538ad08af8c914ba1144adf74263#file-stringequals-cs

Reply:
Thanks! Let's keep this PR smaller for easier/cleaner backport. @am11 feel free to file a PR to clean that up after

Reply:
> feel free to file a PR to clean that up after

https://github.com/dotnet/runtime/pull/78265